### DISCUSSION TITLE
双活模式，是不是基于数据同步实现的

### DISCUSSION THREAD

**Discussion by liuboAAAAA** [apache/iotdb#8332] [General]
是不是基于数据同步实现的？有相关文档吗？

**Comment by yschengzi** [comment]
不是，暂无文档

**Comment by liuboAAAAA** [comment]
如果我想实现双活模式，有相关指导说明吗？



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年04月07日 14:31 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [apache/iotdb] 双活模式，是不是基于数据同步实现的 (Discussion #8332) |

不是，暂无文档
